extensions/amp-story-shopping/0.1/amp-story-shopping-tag.css

```diff
@@ -13,6 +13,17 @@ amp-story-shopping-tag {
   --i-amphtml-shopping-tag-backdrop-filter: blur(20px);
 }
 
+.amp-story-shopping-product-tag-text {
+  font-family: 'Poppins','Arial', sans-serif !important;
+  font-weight: 600 !important;
+  font-size: 11px !important;
+  line-height: 15.4px !important;
+  font-style: normal;  
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.01em;
```

Comment:
Add `!important` to these properties so that they can't be overridden

Reply:
Does this letter-spacing value make a necessary visual difference? I'm reading [here](https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing#internationalization_concerns) that some written languages (such as Arabic) shouldn't have any letter spacing applied, so I'm wondering if publishers using such languages would encounter issues if we specify the spacing.

@gmajoulet have any thoughts here?

Reply:
Good point, I have removed the line spacing property.